odoo/api.py

```diff
@@ -1064,14 +1066,20 @@ def insert_missing(self, records, field, values):
         if field.translate:
             if records.env.context.get('prefetch_langs'):
                 langs = {lang for lang, _ in records.env['res.lang'].get_installed()} | {'en_US'}
+                _langs = {f'_{l}' for l in langs} if field._lang(records.env).startswith('_') else set()
                 for id_, val in zip(records._ids, values):
                     if val is None:
                         field_cache.setdefault(id_, None)
                     else:
-                        val_all_en = dict.fromkeys(langs, val['en_US'])
-                        field_cache[id_] = {**val_all_en, **val}
+                        if _langs:  # fallback missing _lang to lang if exists
+                            val.update({f'_{k}': v for k, v in val.items() if k in langs and f'_{k}' not in val})
+                        field_cache[id_] = {
+                            **dict.fromkeys(langs, val['en_US']),  # fallback missing lang to en_US
+                            **dict.fromkeys(_langs, val.get('_en_US')),  # fallback missing _lang to _en_US
+                            **val
+                        }
```

Comment:
@rco-odoo 
This blocked the runbot
When `prefetch_langs` while `check_translations` I forgot to fallback the cahce for all _langs, which cause a MissingError in the runbot test